kvdb-rocksdb/src/lib.rs

```diff
@@ -214,7 +215,7 @@ impl Default for DatabaseConfig {
 			max_open_files: 512,
 			memory_budget: HashMap::new(),
 			compaction: CompactionProfile::default(),
-			columns: None,
+			columns: unsafe { NonZeroU32::new_unchecked(1) },
```

Comment:
```suggestion
			columns: NonZeroU32::new(1).expect("1 is not a zero; qed"),
```
prefer no unsafe code even though it's perfectly safe